cloud-init/web.txt

```diff
@@ -42,10 +42,7 @@ write_files:
         capture request header Referer len 200
         capture request header User-Agent len 200
 
-        capture response header Content-Type len 30
-        capture response header Content-Encoding len 10
-        capture response header Cache-Control len 200
-        capture response header Last-Modified len 200
+        capture response header Location len 100
```

Comment:
Why this change? IIRC Logstash is expecting our capture stuff to be configured just so.